UnityPy/helpers/ResourceReader.py

```diff
@@ -1,51 +1,33 @@
 import ntpath
-from ..streams import EndianBinaryReader
-from ..files import File
+from ..files.SerializedFile import SerializedFile, EndianBinaryReader
```

Comment:
I'm a bit touchy with SerializedFile due to recursive import issues.
As long as it's only used for type-hinting it should be done with ``if TYPE_CHECKING``.
EndianBinaryReader should also be imported from streams and not indirectly in this way.